cmdstanpy/model.py

```diff
@@ -1,8 +1,8 @@
 import os
-import subprocess
+import subprocess32 as subprocess
```

Comment:
I'd protect the `subprocess32` import in a try-catch so that it defaults to stdlib when user doesn't have the 32 variant installed, in part because author doesn't test on Windows etc

Reply:
Yes, this needs better fix. Python 2.7 doesn't have timeout, so I added this for tmp fix. I can put it in try-except